src/main/scala/http4s/extend/syntax/ByNameNaturalTransformationSyntax.scala

```diff
@@ -3,9 +3,7 @@ package http4s.extend.syntax
 import cats.MonadError
 import http4s.extend.ByNameNaturalTransformation.~~>
 
-import scala.language.higherKinds
```

Comment:
this is still needed here right ? With that it will work also if the flag is not set in the ide

Reply:
I'll add that back